targets/TARGET_Cypress/TARGET_PSOC6/cy_serial_api.c

```diff
@@ -0,0 +1,311 @@
+/***************************************************************************//**
+* \file cy_serial_api.c
+*
+* \brief
+* 
+********************************************************************************
+* Copyright (c) 2018-2019 Cypress Semiconductor.  All rights reserved.
+* You may use this file only in accordance with the license, terms, conditions, 
+* disclaimers, and limitations in the end user license agreement accompanying 
+* the software package with which this file was provided.
+********************************************************************************/
```

Comment:
This header here should be fixed (the license - the previous files were apache, why not this one).

Reply:
Old license header was added by mistake, fixed in https://github.com/ARMmbed/mbed-os/pull/10692/commits/b9f7836ac2bbb48acf4336ba1be5aa0da41cb990#diff-0c19498117fa0dc69058c923815fb238R2